src/remoteConnector.ts

```diff
@@ -429,6 +430,16 @@ export default class RemoteConnector extends Disposable {
 		}
 	}
 
+	private async getIdentityFilePaths(gitpodHost: string) {
+		const list = await checkDefaultIdentityFiles();
+		const sshConfiguration = await SSHConfiguration.loadFromFS();
+		const hostConfiguration = sshConfiguration.getHostConfiguration(new URL(gitpodHost).host);
+		if (!!hostConfiguration.IdentityFile) {
+			list.unshift(hostConfiguration.IdentityFile);
```

Comment:
from the docs `hostConfiguration.IdentityFile` is an array, can it also be a string? if so we should validate it and make it an array 
We also need to validate if the file exists

Reply:
Note: With [ssh config doc](https://www.ssh.com/academy/ssh/config), `IdentityFile` is only a file, but with [ssh-config](https://github.com/cyjake/ssh-config#compute-parameters-by-host) module, it'll return an array after `compute`

Reply:
yeah that's what I meant about [ssh-config](https://github.com/cyjake/ssh-config#compute-parameters-by-host) module, it's already fixed when I did my changes